internal/pkg/service/common/etcdop/iterator/option.go

```diff
@@ -38,3 +39,12 @@ func WithPageSize(v int) Option {
 		c.pageSize = v
 	}
 }
+
+func WithRev(v int64) Option {
+	if v <= 0 {
+		panic(errors.New("revision must be greater than 0"))
+	}
+	return func(c *config) {
+		c.revision = v
+	}
+}
```

Comment:
Ked chceme mat atomicitu pri nacitvani viacerych prefixov, tam musime vediet zadat revision externe.
Takto mozeme iterovat receivers a exports, a mat istotu, ze hodnoty dostavame v tom isto "snapshote".

Pouzil som to na jednom mieste, ... doriesime to, ked sa dokoncia ostatne veci, ktore robi Honza.